### PR TITLE
Fix incorrect header depth in OnlineOverlay

### PR DESCRIPTION
**Pull request by EVAST9919**
Overlooked and removed in https://github.com/ppy/osu/pull/11527/commits/b5fa9508006c76decac0752a6bdaf47598196d4d commit.
master:
master
this pr:
pr

### CODE REVIEW

File: osu.Game/Overlays/OnlineOverlay.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ protected OnlineOverlay(OverlayColourScheme colourScheme)
                         Direction = FillDirection.Vertical,
                         Children = new Drawable[]
                         {
-                            Header,
+                            Header.With(h => h.Depth = -float.MaxValue),
```
Comment (frenzibyte) [inline osu.Game/Overlays/OnlineOverlay.cs:35]: That negative max value really feels like what caused the confusion. 
```suggestion
                            Header.With(h => h.Depth = float.MinValue),
```

**Comment by peppy** [comment]
Why not just move it below the content in the initialization?

**Comment by frenzibyte** [comment]
Yeah I was gonna suggest that at first, but I've noticed they're inside a fill flow container, so header would be moved all the way below content that way

Was gonna also suggest using a `ReverseChildIDFillFlowContainer`, which should work fine, but felt like it's too much.

**Comment by peppy** [comment]
Right, missed that detail. I think it's fine as is.

**Review by peppy — APPROVED** [review]

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**